website/pages/en/sps/sps-intro.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Introduction
```

Comment:
Also a nitpick, but for SEO, we might want a more specific H1 than "Introduction". The user can know that it's related to Substreams-powered Subgraphs by looking at the nav, but I think for SEO we need more than that. How about this:
```suggestion
title: Introduction to Substreams-powered Subgraphs
```
and then we can change `'sps-intro': ''` in `_meta.js` to `'sps-intro': 'Introduction'` to have a shorter title in the nav.